Source/CarthageKit/Xcode.swift

```diff
@@ -932,7 +932,7 @@ public func buildScheme(scheme: String, withConfiguration configuration: String,
 							argsForBuilding.bitcodeGenerationMode = .Bitcode
 						}
 
-						var buildScheme = xcodebuildTask(["clean", "build"], argsForBuilding)
+						var buildScheme = xcodebuildTask("build", argsForBuilding)
```

Comment:
Given how non-resiliant Xcode is to bad incremental compilation, what would a user have to do to force a clean? I wonder if we should add `carthage clean`